### PR TITLE
Additional review

### PR DESCRIPTION
**Pull request by qdequele**

### CODE REVIEW

File: guides/advanced_guides/prefix.md [hunk 1]
```diff
@@ -16,19 +16,18 @@ Given a set of words in a dataset:
...
-
+query: `s`:
+response:
 - `show`
 - `shine`
 
-but not
-
-- `movies`
-- `musical`
```
Comment (bidoubiwa) [inline guides/advanced_guides/prefix.md]: Why did you remove this?

**Comment by bidoubiwa** [comment]
Everything else is great. Thanks a lot.

File: guides/advanced_guides/prefix.md (hunk 1 above)
Comment (qdequele) [reply guides/advanced_guides/prefix.md]: Look at the entire file, this part seems obvious to me. What do you think about it?

Comment (bidoubiwa) [reply guides/advanced_guides/prefix.md]: I agree that from our perspective this is obvious. The reason why I put it is that you generally look at examples either to reassure your knowledge either because you did not understand after reading the theoretical part. So I thought by adding this very-obvious example people will 100% be sure to understand the idea.
